examples/chef/common/chef-rvc-operational-state-delegate.cpp

```diff
@@ -17,17 +17,31 @@
  */
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app/util/config.h>
-
-#ifdef ZCL_USING_OPERATIONAL_STATE_RVC_CLUSTER_SERVER
 #include <chef-rvc-operational-state-delegate.h>
+#include <platform/CHIPDeviceLayer.h>
 
+#ifdef MATTER_DM_PLUGIN_RVC_OPERATIONAL_STATE_SERVER
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::OperationalState;
 using namespace chip::app::Clusters::RvcOperationalState;
+using chip::Protocols::InteractionModel::Status;
+
+static RvcOperationalState::Instance * gRvcOperationalStateInstance = nullptr;
+static RvcOperationalStateDelegate * gRvcOperationalStateDelegate   = nullptr;
 
-CHIP_ERROR GenericOperationalStateDelegateImpl::GetOperationalStateAtIndex(size_t index, GenericOperationalState & operationalState)
+static void onOperationalStateTimerTick(System::Layer * systemLayer, void * data);
+
+DataModel::Nullable<uint32_t> RvcOperationalStateDelegate::GetCountdownTime()
+{
+    if (mCountDownTime.IsNull())
+        return DataModel::NullNullable;
+
+    return DataModel::MakeNullable((uint32_t) (mCountDownTime.Value() - mRunningTime));
```

Comment:
`mCountDownTime` should be called `mPhaseDuration`, since it's the total duration of the phase, which is the initial value of countdown time.

Also, we must prevent underrun if mRunningTIme >= mPhaseDuration.